app/src/main/java/com/xmartlabs/taskloans/data/common/DataHelper.kt

```diff
@@ -10,7 +10,11 @@ object DataHelper {
   suspend fun <T> mapServiceError(errorMapper: (HttpException) -> Exception, serviceCall: suspend () -> T): T = try {
     serviceCall()
   } catch (e: Exception) {
```

Comment:
I'd change the variable name, maybe exception?

Reply:
i think it would be usefull